### ISSUE TITLE
@lenml/llama2-tokenizer-vocab-emoji listed as dependency and is not in the npm registry

### ISSUE THREAD

**Issue by Mgonand** [lenML/llama2-tokenizer.js#2]
When I execute `pnpm i @lenml/llama2-tokenizer` I am getting this error:
```
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@lenml%2Fllama2-tokenizer-vocab-emoji: Not Found - 404

This error happened while installing the dependencies of @lenml/llama2-tokenizer@1.1.1

@lenml/llama2-tokenizer-vocab-emoji is not in the npm registry, or you have no permission to fetch it.
```

Can you update the dependencies? Thank you in advance.

**Comment by zhzLuke96** [comment]
my bad. This issue is fixed in @lenml/llama2-tokenizer@1.1.2

**Comment by Mgonand** [comment]
Awesome. Is working now. Thanks
